app/components/RoomList/index.js

```diff
@@ -339,6 +346,20 @@ class RoomList extends Component {
 									onClick={() => this.setState({ showRoomDeleter: true })}
 								/>
 							</Button.Group>
+							<span style={{ marginLeft: 16 }}>Status</span>
+							<Select
+								mode="multiple"
+								style={{ marginLeft: 16, width: '30%' }}
```

Comment:
nit: be consistent with quote usage throughout the app. I think we mostly use single quotes